### ISSUE TITLE
Bug with a staff for Invaders. And buffs tank KOBS

### ISSUE THREAD

**Issue by Agnus94** [WarEmu/WarBugs#17389]
Hello dear developers and administration of the game, I found a fairly serious bug that caused me to disappear more than 60 pieces of Invader medallions. The bug is that when you want to view the staff for Invader and put it in the inventory to find out exactly what is missing for it, it writes that there are not enough things, and then takes the Invader medallions. Please check it and fix it, this is a serious bug that interferes with the game, and a lot of people may lose a lot of difficult rewards. I also want to tell you about the bug on the kobs tank, which can bafnut 2 BAFs and they work together and keep giving additional stats from 2 BAFs 10-20 seconds. I hope you can return the lost invader medallions to Me by checking the logs, and give me a good reward for helping me find errors in the game. Thanks.
![посох](https://user-images.githubusercontent.com/68962938/88892502-8b52de00-d24d-11ea-9534-7fa8a73e4ce1.png)

**Comment by Kaelang** [comment]
Hi - we don't return any lost items or currency.

And I'm struggling to work out what you mean, you place the item in your inventory, i.e. purchase it, but it takes the currency and doesn't give you the item?

Also please open a second report for the second issue.

**Comment by Agnus94** [comment]
Yes, when you put it in the inventory, it takes invaders, despite the fact that the item is not purchased because there are not enough medallions, and it takes Them.

**Comment by Agnus94** [comment]
This is quite a serious problem, many can like me view it in the inventory, and because of this staff will take away medallions Invader

**Comment by Kaelang** [comment]
But how does it take your medallions if you don't have enough medallions?

**Comment by Kaelang** [comment]
How many does it take?

**Comment by Agnus94** [comment]
60 Invaders this bug took it from me

**Comment by Kaelang** [comment]
I'm unable to reproduce this .
